moveit_plugins/moveit_ros_control_interface/src/controller_manager_plugin.cpp

```diff
@@ -354,7 +358,7 @@ class MoveItControllerManager : public moveit_controller_manager::MoveItControll
     }
     return true;
   }
-  typedef boost::shared_ptr<MoveItControllerManager> Ptr;
+  typedef MoveItControllerManagerPtr Ptr;
```

Comment:
This is a source file and no exposed interface. Wouldn't it make sense to get rid of the `Ptr` typedef here altogether?